build.properties

```
@@ -90,17 +90,17 @@ javadoc.top=<div style="padding: 6px; text-align: center; font-size: 80%;">This
 jdk.docs.version=24
 jfx.build.jdk.version=24.0.1
 jfx.build.jdk.buildnum=9
-jfx.build.jdk.version.min=23
-jfx.build.jdk.buildnum.min=37
-jfx.jdk.target.version=23
+jfx.build.jdk.version.min=24
+jfx.build.jdk.buildnum.min=36
```

```suggestion
jfx.build.jdk.buildnum.min=38
```

Not 100% sure but I think the number should _increase_ and not _decrease_

---

No. Build numbers are only meaningful within a specific release. JDK 23 GA was build 37 (jdk-23+37). JDK 24 GA was build 36 (jdk-24+36).